app.css

```diff
@@ -1,3 +1,10 @@
 body {
-  background: purple;
+  background: white;
```

Comment:
Background defaults to white, but we usually use `margin: 0` to get rid of the whitespace that is around the whole page.

Also, don't forget the `box-sizing` reset.